### PR TITLE
Announce agenda for the minisummit

### PR DESCRIPTION
**Pull request by marmarek**

### CODE REVIEW

**Comment by marmarek** [comment]
@andrewdavidwong this likely needs some more blurb text etc. I'm also not sure about category - should be annoucement or talks?

**Comment by marmarek** [comment]
PipelineRetry

**Comment by marmarek** [comment]
PipelineRetry

File: 2021-07-27-minisummit-agenda.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Upcomming Qubes OS - 3mdeb mini-summit"
```
Comment (pietrushnic) [inline 2021-07-27-minisummit-agenda.md:3]:
```suggestion
title: "Upcomming Qubes OS - 3mdeb mini-summit 2021"
```

**Comment by marmarek** [comment]
testdeploy

**Comment by marmarek** [comment]
PipelineRefresh

**Comment by marmarek** [comment]
testdeploy

File: 2021-07-27-minisummit-agenda.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+### Wyng-backups - revertible local and remote known safe QubesOS states (including dom0)
+
+[Wyng-backups](https://github.com/tasket/wyng-backup) is an incremental
+backup/restore tool for LVMs. For QubesOS, this means even dom0 can be reverted
+to a known safe state; locally or remotely, applying changes only. Deep dive
+into possibilities of wyng-backups to deploy and maintain up to date,
+revertible states of QubesOS base systems.
+
+### RTM and Secure Boot for VMs
```
Comment (pietrushnic) [inline 2021-07-27-minisummit-agenda.md:67]:
```suggestion
### SRTM and Secure Boot for VMs
```

**Comment by marmarek** [comment]
testdeploy

**Comment by marmarek** [comment]
PipelineRefresh

**Comment by marmarek** [comment]
testdeploy

**Comment by andrewdavidwong** [comment]
Revision pushed.

> @andrewdavidwong this likely needs some more blurb text etc.

I'm afraid I don't know enough about this event to create more content. Can you provide a description of what you're looking for here?

> I'm also not sure about category - should be annoucement or talks?

Why not both? :D

**Comment by andrewdavidwong** [comment]
PipelineRefresh

**Comment by andrewdavidwong** [comment]
testdeploy

**Comment by marmarek** [comment]
PipelineRefresh

**Comment by marmarek** [comment]
testdeploy

**Comment by marmarek** [comment]
PipelineRetry

**Comment by andrewdavidwong** [comment]
What's the difference between `PipelineRefresh` and `PipelineRetry`?

**Comment by marmarek** [comment]
Refresh only fetches job status from gitlab, Retry schedules new build (regardless of the status). I used Retry this time, because I noticed the previous one used outdated repo content (before your changes).

**Comment by marmarek** [comment]
testdeploy

**Comment by marmarek** [comment]
I think it looks fine now. We could add some reference to previous events, but I'm not sure if it's really necessary.
@pietrushnic do you want to add something here, or should we publish it now?

**Comment by pietrushnic** [comment]
Past events presentation and videos are here [2019](https://3mdeb.com/events/#Qubes-OS-and-3mdeb-minisummit) and [2020](https://3mdeb.com/events/#Qubes-OS-and-3mdeb-minisummit2020). Letting people know this is not first time has its own value, but it is not necessary - other then that blog post looks good to me.

**Comment by marmarek** [comment]
PipelineRetry

**Comment by marmarek** [comment]
PipelineRetry

**Comment by marmarek** [comment]
PipelineRetry+head

**Comment by marmarek** [comment]
PipelineRetry

**Comment by marmarek** [comment]
testdeploy

File: 2021-07-27-minisummit-agenda.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+This talk is the continuation of the Qubes OS mini-summit presentation "SRTM
+for Qubes OS VMs", where the theoretical background of the Static Root of Trust
+was presented and discussed. In this presentation, we will practically approach
+SRTM and Secure Boot for VMs. We will also explore potential use cases for
+self-decrypting storage and signed kernels using safeboot. Finally, we will
+discuss how to introduce this and other security mechanisms in Qubes OS.
+
+### "Usability Within A Reasonably Secure, Multi-Environment System" by Nina Alter
+
+@TODO
```
Comment (andrewdavidwong) [inline 2021-07-27-minisummit-agenda.md:94]: This was a duplicate of the prior entry.

File: 2021-07-27-minisummit-agenda.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+We may all relate to having a rough time when starting using Qubes --- be that
+because we're coming from Windows and everything is different or because we
+come from Linux and many things don't work like we expect them to. Apart from
+the usual challenges of going into a different system, Qubes has the additional
+one of requiring a fundamentally different way of thinking about your computer
+(a hypervisor mental-model). Smoothing out this transition is particularly
+important as Qubes aims to target vulnerable populations that are less
+technically inclined and have less time to explore and read the documentation.
+
+The solution proposed by deeplow is to implement an integrated onboarding
```
Comment (andrewdavidwong) [inline 2021-07-27-minisummit-agenda.md:67]: We shouldn't link to only one person's name. If we're going to link to anyone's name, we should link to everyone's name. Can go either way; up to you.

Comment (andrewdavidwong) [reply 2021-07-27-minisummit-agenda.md:67]: If we do, probably best to do it in the tables above.

**Review by andrewdavidwong — CHANGES_REQUESTED** [review]

File: 2021-07-27-minisummit-agenda.md (hunk 4 above)
Comment (marmarek) [reply 2021-07-27-minisummit-agenda.md:67]: I'm fine with dropping (I copied it from the original abstract we've got).

**Comment by marmarek** [comment]
PipelineRetry

File: 2021-07-27-minisummit-agenda.md (hunk 3 above)
Comment (marmarek) [reply 2021-07-27-minisummit-agenda.md]: fixed

**Comment by marmarek** [comment]
PipelineRetry

**Comment by marmarek** [comment]
testdeploy

**Comment by marmarek** [comment]
testdeploy

**Pull request merged**